mysql-test/t/MDEV14474-I_S_check_constraint.test

```diff
@@ -0,0 +1,57 @@
+--echo #
+--echo # Task MDEV14474: Create INFORMATION_SCHEMA.CHECK_CONSTRAINTS
```

Comment:
Usually we don't write it as
`Task MDEV14474: Create INFORMATION_SCHEMA.CHECK_CONSTRAINTS`

Instead we write it like so:
`MDEV-14474: Create INFORMATION_SCHEMA.CHECK_CONSTRAINTS`